frontend/src/components/LoginComponent.js

```diff
@@ -0,0 +1,174 @@
+import React, { useState } from "react";
+import { Button, Row, Col, Label } from "reactstrap";
+import StudentComponent from "./StudentComponent";
+import { Control, LocalForm, Errors } from "react-redux-form";
+import { Switch, Route, Redirect } from "react-router-dom";
+import AdminComponent from "./AdminComponent";
+import { withRouter } from "react-router-dom";
+
+const required = (val) => val && val.length;
+
+function LoginComponent(props) {
+  const [login, setLogin] = useState({
+    approved: true,
+    type: "admin"
+  });
+  const [userData, setUserData] = useState({});
+
+  const handleSubmit = (values) => {
+    if (login.type === "admin" && login.approved === true){
+      props.history.push("/admin");
+    } 
+    else if (login.type === "student" && login.approved === true){
+      
+      props.history.push("/student");
+    } else{
+      alert('Incorrect Username or Password Entered')
+    }
+
+    /*setLogin({
```

Comment:
is this needed?

Reply:
Still testing to see if i can get a response from the backend. After a proper response is confirmed the top part can be removed and the commented part can be used as the actual system code.